gcp_variant_transforms/bq_to_vcf.py

```diff
@@ -153,6 +151,26 @@ def _bigquery_to_vcf_shards(
          | vcfio.WriteVcfDataLines())
 
 
+def _form_customized_query(known_args):
+  # type: (argparse.Namespace) -> str
+  """Returns a customized query for the interested regions."""
+  base_query = _BASE_QUERY_TEMPLATE.format(INPUT_TABLE='.'.join(
+      bigquery_util.parse_table_reference(known_args.input_table)))
+  if not known_args.genomic_regions:
+    return base_query
+
+  conditions = []
+  for region in known_args.genomic_regions:
+    ref, start, end = genomic_region_parser.parse_genomic_region(region)
+
+    conditions.append('{}=\'{}\' AND {}>={} AND {}<={}'.format(
```

Comment:
Consider using named parameters here (e.g. {REFERENCE_NAME_ID}, {REFERENCE_NAME_VALUE}, etc) as that would make it easier to read.

Reply:
Done.